input/pagecontent/reference.md

```diff
@@ -602,7 +605,14 @@ For locally-defined extensions, using the slice name is the simplest choice. For
 
 #### Caret Paths
 
-FSH uses the caret (^) symbol to access elements of definitional items corresponding to the current context. Caret paths SHALL be accepted in the following FSH items: Profile, Extension, <span style="background-color: #fff5e6;">{%include tu.html%} Logical, Resource</span>, ValueSet, and CodeSystem. Caret syntax SHOULD be reserved for situations not addressed through [FSH Keywords](#fsh-items) or external configuration files. Examples of elements that require the caret syntax include StructureDefinition.experimental, StructureDefinition.abstract and ValueSet.purpose. The caret syntax also provides a simple way to set metadata attributes in the ElementDefinitions that comprise the snapshot and differential tables (e.g., short, meaningWhenMissing, and various [slicing discriminator properties](#step-1-specify-the-slicing-logic)).
+FSH uses the caret (^) symbol to access elements of definitional items corresponding to the current context. Caret paths SHALL be accepted in the following FSH items: Profile, Extension, <span style="background-color: #fff5e6;">{%include tu.html%} Logical, Resource</span>, ValueSet, and CodeSystem. Caret syntax SHALL NOT be used with the following paths, because they are otherwise addressed through FSH rules:
```

Comment:
I'm not sure that "because they are otherwise addressed through FSH rules" quite captures it.  If we prohibited all things that are "otherwise addressed by FSH rules" then we'd have SHALL NOT for `^name`, `^id`, `^title`, element `^min`, element `^max`, etc) -- but I think (somewhere) we say you SHOULD NOT use caret rules for things already covered by other FSH rules.

How do you feel about:

> Caret syntax SHALL NOT be used with the following paths because the exact order and distribution of items in these paths may vary between implementations.

In this case, it might also make sense to remove the `[n]` from the paths.

Reply:
Also, would this be a substantive change requiring re-ballot?  If so, we may just want to scrap it.

Reply:
I view it as a minor correction to what we had in the ballot. Should be ok.

Reply:
Fix made.